modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -266,9 +248,39 @@ function VocabularyQuery(props) {
           </MenuItem>
           );
       });
-      showUserEntry && suggestions.push(<Divider key="divider"/>);
     }
+
+    var allRequestsFailed = Object.keys(statuses).filter(vocab => !statuses[vocab]).length == 0;
+    var allRequestsSucceded = Object.keys(statuses).filter(vocab => statuses[vocab]).length == 0;
+
+    if (!allRequestsSucceded) {
+      if (suggestions.length > 0 && !allRequestsFailed) {
+        suggestions.push(<Divider key="error-divider"/>);
+      }
```

Comment:
Let's simplify the whole syntax actually
```suggestion
     suggestions.length > 0 && suggestions.push(<Divider key="error-divider"/>);
```

Reply:
Done in cb2b5e9